React/my-app/src/screens/login/layout.js

```diff
@@ -1,28 +1,29 @@
 import React, { Component } from "react";
 import "./styles.css";
 import wBooks from "../../assets/photos/wBooks.png";
-import { Link } from "react-router-dom";
+import Input from "../../components/Input";
 import ErrorMsg from "../../components/ErrorMsg";
-import { Redirect } from "react-router-dom";
-import { WBOOKS, INGRESAR, EMAIL, PASSWORD } from "./strings";
+import { Link } from "react-router-dom";
+import { WBOOKS, INGRESAR, EMAIL, PASSWORD, SIGN_UP } from "./strings";
+import Logo from "../../components/Logo";
 
 const Login = ({ onSubmit, emailError, passwordError }) => (
   <div className="login_box">
-    <div className="logo">
-      <img className="book" src={wBooks} />
-      <h1 className="big_title">{WBOOKS}</h1>
-    </div>
+    <Logo picture={wBooks} text={WBOOKS} />
     <form className="login_form" onSubmit={onSubmit}>
-      {EMAIL}
-      <input name="email" className="data" type="email " />
+      <Input title={EMAIL} name="email" type="email" />
       <ErrorMsg message={emailError} />
-      {PASSWORD}
-      <input name="password" className="data" type="Password" />
+      <Input title={PASSWORD} name="password" type="Password" />
       <ErrorMsg message={passwordError} />
       <button className="enter" type="submit">
         {INGRESAR}
       </button>
     </form>
+    <Link to="/sign_up">
+      <button className="sign_up" type="button">
```

Comment:
or use a Link with a label styled or use a button with a onClick function. I think that both is redundant.